app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -90,6 +90,10 @@ CONFIG_LLEXT_STORAGE_WRITABLE=y
 CONFIG_MODULES=y
 CONFIG_LIBRARY_BASE_ADDRESS=0xa0688000
 
+# Modular by default components. Compile-tested but usually not needed at run-time
```

Comment:
Hmm, why are Intel board file changes and generic module code changes in the same commit? Especially as the config change is done to only some targets (e.g. not even all ace targets), so a commit explaining the reasoning would be nice.